tests/e2e/tutorials/sleepers.js

```diff
@@ -18,7 +19,7 @@ const {
 const templateName = "Sleepers";
 
 async function runTutorial() {
-  const tutorial = new tutorialBase.TutorialBase(url, user, pass, newUser, templateName);
+  const tutorial = new tutorialBase.TutorialBase(url, user, pass, newUser, templateName, enableDemoMode);
```

Comment:
Please, move the args processing step to the TutorialBase or add the same thing in the Mattward tutorial

Reply:
good point.